lib/workers/repository/process/lookup/timestamps.ts

```diff
@@ -0,0 +1,57 @@
+import { DateTime } from 'luxon';
+import type {
+  Release,
+  ReleaseResult,
+} from '../../../../modules/datasource/types';
+import type { VersioningApi } from '../../../../modules/versioning/types';
+import { asTimestamp } from '../../../../util/timestamp';
+
+export function calculateLatestReleaseBump(
+  versioningApi: VersioningApi,
+  releaseResult: ReleaseResult,
+): ReleaseResult {
+  let highestRelease: Release | undefined;
+  for (const release of releaseResult.releases) {
+    if (!highestRelease) {
+      highestRelease = release;
+      continue;
+    }
+
+    try {
+      if (
+        versioningApi.isGreaterThan(release.version, highestRelease.version)
+      ) {
+        highestRelease = release;
+        continue;
+      }
+      // eslint-disable-next-line no-empty
+    } catch /* v8 ignore next */ {}
```

Comment:
shouldn't we at least log a trace message if it fails? so you can remap it for debugging purposes?

it's also easily testable because you can pass a mock versioning which throws